bitnami/grafana-operator/templates/grafana.yaml

```diff
@@ -38,6 +38,8 @@ spec:
     {{- if .Values.commonAnnotations }}
     annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 6 }}
     {{- end }}
+  serviceAccount:
+    {{- include "common.images.pullSecrets" (dict "images" (list .Values.operator.image .Values.grafana.image .Values.grafanaPluginInit.image) "global" .Values.global) | nindent 4 }}
```

Comment:
Why did you add the image pull secrets under the "serviceAccount" section?

Reply:
@marcosbc Suggested I give it a try.  See https://github.com/bitnami/charts/issues/5762 for more details.

Reply:
Apparently it's sth supported. See https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-image-pull-secret-to-service-account
I wasn't aware of this feature.